src/routes/windows-protection/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Layout from '../../components/Layout';
+import WindowProtection from './WindowProtection';
+
+const title = 'Window protection';
+
+export default function action() {
+  return {
+    chunks: ['windows-protection'],
```

Comment:
Sometimes windows protection, sometimes window protection, be careful, I'd suggest using same variant in all places